admin/allow-services.rst

```diff
@@ -47,36 +52,65 @@ The following examples show how to set the ``HTTP_PROXY`` and ``HTTPS_PROXY`` en
 
 .. tabs::
 
-   .. code-tab:: powershell Windows
+   .. code-tab:: bash Linux (Systemd)
+
+      # Add proxy settings to the environment for the installer script
+
+      sudo sed -i "$ a http_proxy=http://<proxy.address:port>" /etc/environment
+      sudo sed -i "$ a https_proxy=http://<proxy.address:port>" /etc/environment
+      sudo sed -i "$ a no_proxy=<address>" /etc/environment
+
+      # Add proxy configuration to the service-proxy.conf
+      # file in /etc/systemd/system/splunk-otel-collector.service.d/
+
+      sudo touch /etc/systemd/system/splunk-otel-collector.service.d/service-proxy.conf
```

Comment:
No need to create the file since it'll be created in the subsequent command. But I think the directory should first be created, just in case:
```suggestion
      sudo mkdir -p /etc/systemd/system/splunk-otel-collector.service.d/
```